back/src/Guide/Database/Schema.hs

```diff
@@ -211,7 +179,7 @@ v0_createTableUsers = HS.sql $ toByteString [text|
 -- | Create table @pending_edits@, storing users' edits and metadata about
 -- them (who made the edit, when, etc).
 v0_createTablePendingEdits :: Session ()
-v0_createTablePendingEdits = HS.sql $ toByteString [text|
+v0_createTablePendingEdits = HS.sql [r|
```

Comment:
This table should also use `timestamptz` instead of `timestamp`. Please be more careful when grepping.